### ISSUE TITLE
inference time

### ISSUE THREAD

**Issue by wldeephi** [sacmehta/ESPNet#11]
@sacmehta , thanks very much for your work, how about the inference time for one image with input_size=1024x512 ? I saw the time is 0.0089s per image shown in the cityscapes benchmark, but I get the time is 0.132s with input_size=1024x512 on GPU, and my GPU is Titanx 1080.

**Comment by sacmehta** [comment]
Could you please try to measure the model execution time only i.e. without image loading and image writing time?

Note: For the very first iteration, inference  time will be low.

**Comment by MrLinNing** [comment]
Hi, @sacmehta 
I test the 1024x512 shape of image on TITAN xp, and the FPS is about 100.
Your paper report 112 fps on the NVIDIA TitanX.
Do you have any skill, like absorving the batchnorm layers to conv layer? 
Besides, my cuda version is 9.0 and cudnn is 7.1.

**Comment by sacmehta** [comment]
No, we don’t combine batch norm withconv layers. Are you discarding the first iteration?
